data/bangs.json

```diff
@@ -48228,7 +48228,7 @@
     "s": "Jisho (kanji search)",
     "d": "jisho.org",
     "t": "jik",
-    "u": "https://jisho.org/search/%23kanji%20{{{s}}}",
```

Comment:
This one too. (URL template requires `#` to be encoded so it is not interpreted as a fragment identifier.)